src/modules/Navbar/Navbar.css

```diff
@@ -0,0 +1,89 @@
+.NavbarItems {
+    background: linear-gradient(90deg, rgb(0, 0, 0) 0%, rgb(6, 6, 7) 100%);
+    height: 100px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: 1.2rem;
+}
+
+
+.navbar-logo{
+    color:white;
```

Comment:
Try not to use pure white and black colors. This can be bad for people's eyes. Try lowering the white color to #EEEEEE instead of using the default white (#FFFFFF).